src/autoscaler/Makefile

```
@@ -146,3 +150,28 @@ clean:
 	@rm --force --recursive 'fakes'
 	@rm --force --recursive 'vendor'
 	@rm --force --recursive "${openapi-generated-clients-and-servers-dir}"
+
+.PHONY: mta-deploy
+mta-deploy: mta-build build-extension-file
+	$(MAKE) -f metricsforwarder/Makefile set-security-group
+	$(MAKE) -f metricsforwarder/Makefile stop-metricsforwarder-vm
+	@echo "Deploying with extension file: $(EXTENSION_FILE)"
+	@CF_TRACE=true cf deploy mta_archives/*.mtar -f -e $(EXTENSION_FILE)
```

Do we need to trace?
```suggestion
	@cf deploy mta_archives/*.mtar -f -e $(EXTENSION_FILE)
```